python/ml4ir/base/config/parse_args.py

```diff
@@ -302,6 +302,27 @@ def define_args(self):
             help="Whether to track model performance and hyperparameters.",
         )
 
+        self.add_argument(
+            "--gl_2_clicks",
```

Comment:
@mohazahran Thinking more about this, I feel it makes sense to add this to the feature_config yaml under the label feature. What do you think?

Reply:
That's possible. However, that requires supporting the conversion to clicks to other data format (CSV and TFrecords) which are not currently supporting this.